src/analyze/analyze-capability.c

```diff
@@ -5,10 +5,23 @@
 #include "cap-list.h"
 #include "capability-util.h"
 #include "format-table.h"
+#include "parse-util.h"
+
+static int table_add_capability(Table *table, int c) {
+        int r;
+
+        r = table_add_many(table,
+                           TABLE_STRING, capability_to_name(c) ?: "cap_???",
+                           TABLE_UINT, c);
+        if (r < 0)
+                return table_log_add_error(r);
+        return r;
```

Comment:
just do 'return 0` here, you have dealt with the error handling, no need to return the return code you received.